lib/custom-api/user.js

```diff
@@ -22,9 +23,20 @@ const defaultUser = {
   type: 'sysadmin'
 }
 
+const handleErrorAndBadRequest = (err, badRequest, res) => {
+  if (err) {
+    logger.error(err)
+    res.status(err.statusCode).send(err.message)
```

Comment:
There's a missing return here. I'm not sure if both err and badRequest can be truthy at the same time, but I think it's safest to prevent the possibility of trying to send multiple responses.

Reply:
Thanks @bausmeier well spotted!, Nope they cant be truthy at the same and the return is needed.